src/annotator/anchoring/test/xpath-test.js

```diff
@@ -100,14 +100,16 @@ describe('annotator/anchoring/xpath', () => {
         const result = nodeFromXPath(test.xpath, container);
         assert.equal(result.nodeName, test.nodeName);
       });
+    });
 
-      it('it returns the node associated with the XPath when `document.evaluate` throws an error', () => {
-        const result = nodeFromXPath(test.xpath, container);
-        sinon.stub(document, 'evaluate').throws(new Error());
-        const resultFallback = nodeFromXPath(test.xpath, container);
-        assert.equal(result, resultFallback);
-        document.evaluate.restore();
-      });
+    it('returns `null` if XPath evaluation fails', () => {
+      assert.equal(nodeFromXPath('/missing/element', container), null);
+    });
+
+    it('throws if XPath is invalid', () => {
+      assert.throws(() => {
+        nodeFromXPath('not-a-valid-xpath', container);
+      }, /The string '.not-a-valid-xpath' is not a valid XPath expression/);
```

Comment:
The behavior here is the same as before except for the text of the error message that is thrown. Previously it was "wanted child not found".